src/java.desktop/macosx/native/libosxapp/JNIUtilities.m

```diff
@@ -40,11 +40,18 @@
 }
 
 jstring NSStringToJavaString(JNIEnv* env, NSString *str) {
-
     if (str == NULL) {
        return NULL;
     }
-    jstring jStr = (*env)->NewStringUTF(env, [str UTF8String]);
```

Comment:
Probably we should cleanup all usage of [nsstring UTF8String] in the code at some point.

Reply:
Depends on what the source data is.
FWIW in the macOS code there are just 5 others and none of them were touched/added/changed by the JNF work. They are all pre-existing.